content/departments/security/knowledge-base/using-iap.md

```diff
@@ -0,0 +1,27 @@
+# Enabling IAP for TCP Forwarding
+
+To set up a compute instance with SSH access, you must use Google's Identity-Aware Proxy (IAP). This method introduces an authentication layer that requires your Google Workspace credentials before establishing a connection through the TCP port. This strategy ensures that no unwanted SSH services are exposed on our external perimeter.
+
+Our security team has implemented detections that will alert us to any exposed SSH services.
+
+## Requirements
+
+- Compute instance
+- Firewall rules that enable IAP server access to the instance
+
+## Setting Up Firewall Rules
+
+After setting up the instance, you must configure it to accept connections from Google's edge servers to your SSH service. Use this [guide](https://cloud.google.com/iap/docs/using-tcp-forwarding#create-firewall-rule) to create the appropriate firewall rule. While the guide suggests targeting "All instances in the network," you have the option to be more selective by using a network tag.
```

Comment:
I seem to recall we already have a firewall rule for this - would be good to mention the name of this so people don't create new ones

Reply:
Good catch. We don't have a firewall rule for that in every project, but some might. 👍